src/main/java/core/basesyntax/file/FileWriterImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.file;
+
+import exception.FileException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String filePath, String report) {
+        try {
+            Files.writeString(Path.of(filePath), report);
+        } catch (IOException e) {
+            throw new FileException("Can't write data to file: " + filePath);
```

Comment:
Consider including the original exception `e` as a cause in the `FileException` to provide more context about the error. For example: `throw new FileException("Can't write data to file: " + filePath, e);`